test/fixtures/rules/std_rule_cases/L025.yml

```diff
@@ -42,3 +42,12 @@ test_6:
   configs:
     core:
       dialect: bigquery
+
+test_7:
+  # Test ambiguous column reference b/c of BQ STRUCT: https://github.com/sqlfluff/sqlfluff/issues/1079
```

Comment:
b/c?

Reply:
Are you asking why it's ambiguous?

Reply:
I updated the comment to clarify why it's ambiguous.

Reply:
Sorry - I'm asking what b/c means!

Reply:
Oh, it means "because". Lazy typing... I don't know if this is a USA thing or a "me" thing!